client.js

```diff
@@ -13,19 +13,19 @@ const helpMsg = '\n\n' +
     '\n' +
     '***COMMANDS:***\n' +
     '!help' +
-    '> Displays available commands.\n' +
+    '   →   Displays available commands.\n' +
```

Comment:
Ah, ">" is specifically for quoting, it is special discord formatting. I BELIEVE it should look better than a unicode arrow? But I'll look at both and see what I think shortly.

Reply:
Oh, I didn't notice.  Let me know.

Reply:
Yeah, IMO it looks better with the quoting, if you want to change it back 👍

Reply:
Done!